virt/install/runtest.sh

```diff
@@ -821,7 +821,13 @@ EOF
          connected=0
          tries=30
          while [ $connected -eq 0 -a $tries -gt 0 ]; do
-            ping -c 1 -4 $LAB_CONTROLLER && connected=1 && break
+           
+	    if rlIsRHEL '7.2' ; then
+               ping -c 1 $LAB_CONTROLLER && connected=1 && break
+            else
+               ping -c 1 -4 $LAB_CONTROLLER && connected=1 && break
```

Comment:
Hi @ZhonghuaHao!

Thank you for your contribution.
I have 2 things.

- CI failed due to the "Double quote to prevent globbing and word splitting." That's the one thing for the fix.
- I guess this is not the only problem for RHEL 7.2. But we can tell that 7.2- will also have problem. The condition should be updated accordingly.

Reply:
Hi @StykMartin ,
I am so sorry for reply lately.I have solved this two things you mentioned. 
The success commit:  d6c5773ed997e3e0c7797b4bb8fed22c274ea42f.
Could you help to review it again?Thank you very much.